lib/board.rb

```diff
@@ -0,0 +1,50 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+
+  def reset!
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    cells[input.to_i - 1]
+    end
+
+    def full?
+    if  cells.count("X") + cells.count("O") == 9
+           true
+        else
+          false
+        end
+      end
```

Comment:
Instead of breaking into an `if-else`, we could just use Ruby's implicit return and have just the conditional here:
```
def full?
  cells.count("X") + cells.count("O") == 9
end
```